frontend/ChapterList.vue

```diff
@@ -633,7 +633,7 @@ export default Vue.extend({
         this.hideModal();
       }
 
-      this.currentChapter = { ...chapter };
+      this.currentChapter = !chapter ? ({} as Chapter) : { ...chapter };
```

Comment:
these were just some small changes to make TS compiler happy

Reply:
is making it an optional parameter also necessary for the ts compiler? if the old code was working, it seems to imply that the args passed were always non-nullish.
and is this not only necessary when the parameter is optional?

but since this code should hopefully be getting deleted soon, feel free to ignore if it takes non-trivial looking into :)

Reply:
We are calling it in places with only 1 param, so it should be optional unless we change that.